packages/code/src/Code.tsx

```diff
@@ -187,17 +398,45 @@ function Code({
   return (
     <div className={wrapperStyle}>
       {renderedWindowChrome}
-
-      <pre
-        {...(rest as DetailedElementProps<HTMLPreElement>)}
-        className={wrapperClassName}
+      <div
+        className={css`
+          display: flex;
+        `}
       >
-        {showLineNumbers && (
-          <LineNumbers variant={variant} lineCount={lineCount} />
-        )}
+        <pre
+          {...(rest as DetailedElementProps<HTMLPreElement>)}
+          className={wrapperClassName}
+          onScroll={e => {
```

Comment:
I believe the `e` is implied as the first argument so you should just be able to `onScroll={handleScroll}`

Reply:
yup, that's what I had initially. I was playing around with it when I was running in to "Maximum depth exceeded" errors implementing useReducer and never changed it back to `onScroll={handleScroll}`. Fixed now!